teamA/gisuk/app.js

```diff
@@ -31,6 +31,134 @@ app.use(express.json());
     
 app.get("/ping", (req,res) => {
   res.status(200).json({"message" : "pong"});
+});
+
+app.get("/user", async (req,res) => {
+  await appDataSource.query(
+    `SELECT 
+      users.name,
+      users.email,
+      users.profile_image
+    FROM users`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    })
+});
+
+app.get("/user/post/:user_id", async (req,res) => {
+  const user_id = req.params.user_id;
+  await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.profile_image as userProfileImage,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent 
+    FROM posts INNER JOIN users on posts.user_id = users.id 
+    WHERE users.id LIKE ${user_id};`,
+    (err, rows) => {
+      const posting = [];
+      rows.map((el) => {
+        posting.push({
+          postingId : user_id, 
+          postingTitle : el.postingTitle, 
+          postingContent : el.postingContent
+        })
+      })
+    result = {
+      userId : user_id, 
+      userProfileImage : rows[0].userProfileImage, 
+      postings : posting
+    }
+    res.status(200).json({data : result});
+    }
+  )
+})
+
+app.get("/post", async (req,res) => {
+    await appDataSource.query(
+    `SELECT
+      users.id as userId,
+      users.profile_image as userProfileImage,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+    FROM posts INNER JOIN users on posts.user_id = users.id;`,
+    (err, rows) => {
+      res.status(200).json({data :rows});
+  })
+})
+
+app.post("/user", async (req, res, next) => {
+  const { name, email, profile_image, password} = req.body;
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?);`,
+    [ name, email, profile_image, password ]
+    );
+  res.status(201).json({message : "userCreated"});
+})
+
+app.post("/post", async (req, res, next) => {
+  const { title, content, user_id} = req.body;
```

Comment:
```suggestion
  const { title, content, userId} = req.body;
```
윗 코멘트와 비슷한 맥락이라고 볼 수 있겠네요 :)